### PR TITLE
feat:completed HomePage

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
feature:home page completed
Screenshot (1547)
Screenshot (1548)
Screenshot (1549)

### CODE REVIEW

File: Home/home.css [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    border: 2px solid black;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 10;
+    justify-content: space-around;
+    background-color: whitesmoke;
```
Comment (ravikumar0403) [inline Home/home.css:20]: Use variables for color

**Review by zabihshaik — APPROVED** [review]
Everything looks good

File: Home/home.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+.apnatheka_navbar_logo span {
+    font-size: 4rem;
+}
+
+.apnatheka_navbar_logo span+span {
+    color: var(--Drinks-Break-primary-color);
```
Comment (ravikumar0403) [inline Home/home.css:28]: Keep the naming consistent, Use small case for the variable name here

File: Home/home.css (hunk 1 above)
Comment (yashpatel1797) [inline Home/home.css:20]: use variable here also

File: Home/home.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    align-items: center;
+    justify-content: center;
+    gap: 1.5rem;
+    margin: 0 auto;
+}
+
+.inside-banner h1 {
+    font-size: 3rem;
+    font-weight: 900;
+    /* opacity: 0.7; */
```
Comment (ravikumar0403) [inline Home/home.css:91]: remove commented code if not required

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: Home/home.css [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+}
+
+.categories {
+    width: 13rem;
+    position: relative;
+    height: 20rem;
+    padding: var(--padding-large);
+    background-color: whitesmoke;
+    border-radius: 0.5rem;
+    box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px, rgba(0, 0, 0, 0.3) 0px 30px 60px -30px, rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
```
Comment (ravikumar0403) [inline Home/home.css:143]: you can make this entire box-shadow value as a variable as you might need it in multiple places

File: util.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+@import url("https://fonts.googleapis.com/css2?family=Oswald&family=Poppins:ital,wght@0,300;1,900&family=Yanone+Kaffeesatz:wght@400;500&display=swap");
+html {
+    font-family: "Oswald", sans-serif;
+    font-family: "Poppins", sans-serif;
+    font-family: "Yanone Kaffeesatz", sans-serif;
+}
+
+:root {
+    --Drinks-Break-primary-color: #22968c;
```
Comment (ravikumar0403) [inline util.css:46]: You have used a `:root` selector above, So you can add this color as well in the same block

**Review by ravikumar0403 — COMMENTED** [review]
Added a few comments, Rest all looks good

**Pull request merged**